en/python_installation/instructions.md

```diff
@@ -7,7 +7,7 @@ Django is written in Python. We need Python to do anything in Django. Let's star
 
 ### Windows
 
-You can download Python for Windows from the website https://www.python.org/downloads/release/python-351/. After downloading the ***.msi** file, you should run it (double-click on it) and follow the instructions there. It is important to remember the path (the directory) where you installed Python. It will be needed later!
+First check whether your computer is running a 32-bit version or a 64-bit version of Windows at https://support.microsoft.com/en-au/kb/827218. You can download Python for Windows from the website https://www.python.org/downloads/release/python-351/. If your computer is running a **64-bit** version of Windows, download the **Windows x86-64 executable installer**. Otherwise, download the **Windows x86 executable installer**. After downloading the installer, you should run it (double-click on it) and follow the instructions there. It is important to remember the path (the directory) where you installed Python. It will be needed later!
```

Comment:
"https://www.python.org/downloads/windows/" won't be linked to one version in particular. Then, we can tell the reader to click on "Latest Python 3 Release". What do you think?